Compile error on Linux

Encounter the following error message on Linux

```
......
   Compiling dpi v0.1.1
   Compiling simple_logger v5.0.0
   Compiling zvariant_utils v3.2.0
   Compiling enumflags2 v0.7.11
   Compiling wayland-protocols-wlr v0.3.6
   Compiling wayland-protocols-plasma v0.3.6
   Compiling zvariant_derive v5.4.0
   Compiling zvariant v5.4.0
   Compiling sctk-adwaita v0.10.1
   Compiling zbus_names v4.2.0
   Compiling zbus_macros v5.5.0
   Compiling zbus v5.5.0
   Compiling x11rb v0.13.1
   Compiling betrayer v0.4.0 (/home/xubaiw/try-system-tray/betrayer)
error[E0432]: unresolved import `zbus::export::futures_util`
  --> src/platform/linux/mod.rs:13:19
   |
13 | use zbus::export::futures_util::{StreamExt, TryStreamExt};
   |                   ^^^^^^^^^^^^ could not find `futures_util` in `export`

error[E0599]: no method named `then` found for struct `OwnerChangedStream` in the current scope
    --> src/platform/linux/mod.rs:127:66
     |
127  |                     proxy.inner().receive_owner_changed().await?.then(|new_owner| {
     |                     ---------------------------------------------^^^^ method not found in `OwnerChangedStream<'_>`
     |
    ::: /home/xubaiw/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/futures-lite-2.6.0/src/stream.rs:1026:8
     |
1026 |     fn then<F, Fut>(self, f: F) -> Then<Self, F, Fut>
     |        ---- the method is available for `OwnerChangedStream<'_>` here
     |
     = help: items from traits can only be used if the trait is in scope
help: trait `StreamExt` which provides `then` is implemented but not in scope; perhaps you want to import it
     |
1    + use futures_lite::stream::StreamExt;
     |

Some errors have detailed explanations: E0432, E0599.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `betrayer` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
```

It seems the cause is that `zbus` no longer re-exports `futures_utils` in lastest resolved version.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -33,6 +33,7 @@ async-io = "2.3.4"
 parking_lot = "0.12.3"
 flume = "0.11.1"
 png = "0.17.14"
+futures-util = "0.3.31"
 
 [target."cfg(target_os = \"macos\")".dependencies]
 block2 = "0.5.1"

--- a/src/platform/linux/mod.rs
+++ b/src/platform/linux/mod.rs
@@ -7,10 +7,10 @@ use std::sync::atomic::{AtomicU32, Ordering};
 use std::sync::{Arc, OnceLock};
 
 use flume::Sender;
+use futures_util::{StreamExt, TryStreamExt};
 use parking_lot::Mutex;
 use png::{BitDepth, ColorType, Encoder};
 use zbus::{connection, proxy, Task};
-use zbus::export::futures_util::{StreamExt, TryStreamExt};
 
 use crate::error::{ErrorSource, TrayResult};
 use crate::platform::linux::item::StatusNotifierItem;
@@ -24,7 +24,7 @@ static COUNTER: AtomicU32 = AtomicU32::new(1);
 enum TrayUpdate<T> {
     Menu(Menu<T>),
     Tooltip(String),
-    Icon(String)
+    Icon(String),
 }
 
 pub type TrayCallback<T> = Arc<Mutex<dyn FnMut(TrayEvent<T>) + Send + 'static>>;
@@ -35,13 +35,13 @@ pub struct NativeTrayIcon<T> {
     tmp_icon_file: Cell<Option<TmpFileRaiiHandle>>,
     tmp_icon_counter: Cell<u32>,
     _update_task: Task<()>,
-    _register_task: Task<Result<(), zbus::Error>>
+    _register_task: Task<Result<(), zbus::Error>>,
 }
 
 impl<T: Clone + Send + 'static> NativeTrayIcon<T> {
     pub async fn new_async<F>(builder: TrayIconBuilder<T>, callback: F) -> TrayResult<Self>
     where
-        F: FnMut(TrayEvent<T>) + Send + 'static
+        F: FnMut(TrayEvent<T>) + Send + 'static,
     {
         let pid = std::process::id();
         let id = COUNTER.fetch_add(1, Ordering::AcqRel);
@@ -60,7 +60,7 @@ impl<T: Clone + Send + 'static> NativeTrayIcon<T> {
             .name(name.clone())?
             .serve_at(
                 ITEM_PATH,
-                StatusNotifierItem::new(icon.unwrap_or_default(), builder.tooltip.unwrap_or_default(), callback.clone())
+                StatusNotifierItem::new(icon.unwrap_or_default(), builder.tooltip.unwrap_or_default(), callback.clone()),
             )?
             .serve_at(MENU_PATH, DBusMenu::new(builder.menu.unwrap_or_else(Menu::empty), callback))?
             .internal_executor(true)
@@ -110,7 +110,7 @@ impl<T: Clone + Send + 'static> NativeTrayIcon<T> {
                         }
                     }
                 },
-                "event receiver"
+                "event receiver",
             )
         };
 
@@ -124,18 +124,24 @@ impl<T: Clone + Send + 'static> NativeTrayIcon<T> {
         let register_task = {
             conn.executor().spawn(
                 async move {
-                    proxy.inner().receive_owner_changed().await?.then(|new_owner| {
-                        let proxy = &proxy;
-                        let name = &name;
-                        async move {
-                            match new_owner {
-                                Some(_) => proxy.register_status_notifier_item(&name).await,
-                                None => Ok(())
+                    proxy
+                        .inner()
+                        .receive_owner_changed()
+                        .await?
+                        .then(|new_owner| {
+                            let proxy = &proxy;
+                            let name = &name;
+                            async move {
+                                match new_owner {
+                                    Some(_) => proxy.register_status_notifier_item(&name).await,
+                                    None => Ok(()),
+                                }
                             }
-                        }
-                    }).try_collect::<()>().await
+                        })
+                        .try_collect::<()>()
+                        .await
                 },
-                "statusnotifierwatcher watcher"
+                "statusnotifierwatcher watcher",
             )
         };
 
@@ -145,13 +151,13 @@ impl<T: Clone + Send + 'static> NativeTrayIcon<T> {
             tmp_icon_file: Cell::new(tmp_icon_path.flatten()),
             tmp_icon_counter: Cell::new(tmp_icon_counter),
             _update_task: receiver_task,
-            _register_task: register_task
+            _register_task: register_task,
         })
     }
 
     pub fn new<F>(builder: TrayIconBuilder<T>, callback: F) -> TrayResult<Self>
     where
-        F: FnMut(TrayEvent<T>) + Send + 'static
+        F: FnMut(TrayEvent<T>) + Send + 'static,
     {
         async_io::block_on(Self::new_async(builder, callback))
     }
@@ -216,7 +222,7 @@ trait StatusNotifierWatcher {
 pub enum NativeIcon {
     #[allow(dead_code)]
     Path(String),
-    Pixels(Vec<u8>)
+    Pixels(Vec<u8>),
 }
 
 impl NativeIcon {

